docs/development/RELEASE_LIFECYCLE.md

```diff
@@ -8,21 +8,21 @@ https://github.com/zeta-chain/node
 
 ZetaChain uses a dual-branch workflow inspired by GitFlow but adapted for our unique blockchain context. This workflow supports independent release cycles for ZetaCore (consensus-breaking changes) and ZetaClient (non-consensus-breaking features), enabling frequent ZetaClient updates without requiring network-wide consensus upgrades.
 
-Unlike traditional GitFlow where all features eventually merge to a single integration branch, our workflow maintains separate tracks for consensus-breaking changes that require network coordination versus ZetaClient features that can be deployed independently.
-
+Unlike traditional GitFlow where all features eventually merge to a single integration branch, our workflow uses `develop` as the primary integration branch for all changes, while `main` represents the production-ready state for consensus-breaking releases.
 ### Core Branches
 
-**`main`** - Consensus-breaking changes only
+**`develop`** - Primary integration branch
 
-- All changes that affect network consensus
-- Base branch for ZetaCore releases
-- Examples: Protocol upgrades, consensus rule changes, state machine modifications
+- All changes merge here: both consensus-breaking and non-consensus-breaking
+- Base branch for ZetaClient releases
+- Consensus-breaking changes can safely be merged here since they only affect the network when a ZetaCore release is deployed
```

Comment:
nit: maybe should be specified "Most" consensus-breaking changes, if they dont interfere with zetaclient releases, otherwise we should use feature branch, or something like that